lib/Dancer/Plugin/SimpleCRUD.pm

```diff
@@ -1308,11 +1309,17 @@ SEARCHFORM
     }
 
     my @custom_callbacks = ();
+    my @all_cols = map { $_->{COLUMN_NAME} } @$columns;
     for my $custom_col_spec (@{ $args->{custom_columns} || [] } ) {
-        push @custom_callbacks, {
-            column=>$custom_col_spec->{name}, 
-            transform=> ($custom_col_spec->{transform} or sub { return shift;}),
-        };
+        # don't re-show columns that are already displayed
```

Comment:
I do wonder if this is the desired behaviour, or if a custom_column whose name matches that  of a column that would have been in the results should trump the column from the DB, instead, so you could e.g. override a column but keep the same name?